chain/chunks/src/lib.rs

```diff
@@ -397,13 +398,15 @@ pub struct ShardsManager {
     chunk_forwards_cache: SizedCache<ChunkHash, HashMap<u64, PartialEncodedChunkPart>>,
 
     seals_mgr: SealsManager,
+    pub rng_seed: RngSeed,
```

Comment:
All other fields are private, it feels odd to break encapsulation and add a pub one.